modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/change/tracking/spi/display/DLFileVersionCTDisplayRenderer.java

```diff
@@ -62,9 +64,11 @@ public InputStream getDownloadInputStream(
 			DLFileVersion dlFileVersion, String key)
 		throws PortalException {
 
+		PDFProcessor pdfProcessor = (PDFProcessor)_pdfDLProcessor;
```

Comment:
In this case you do not need to create a variable... just cast the parameter in the method call